out/pedalintandem.json

```diff
@@ -0,0 +1,206 @@
+    "location": {
+      "postalAddress": "837/1, 2nd Cross, 7th Main, 2nd Stage, Indiranagar, Bengaluru, Karnataka",
+      "@type": "Place",
+      "address": "Pedal in Tandem bike shop in Indiranagar, Bangalore"
+    },
+    "offers": {
+      "priceCurrency": "INR",
+      "@type": "offers",
+      "workshop": "750.00",
+      "addOn": {
+        "@type": "offers"
+      }
+    },
+    "dates": [
+      {
+        "startDate": "2024-09-08T14:45:00+05:30",
+        "endDate": "2024-09-08T16:45:00+05:30",
+        "availabilityStarts": "2024-09-08T09:30:00+05:30"
+      }
+    ],
+    "duration": 2,
+    "description": "This session will help you understand how your bicycle's drivetrain (or \"gears\") work. You will learn about the configurations available for different types of bikes, how they translate to ride experience, and how you should choose the correct set of configurations for different types of riding, road surfaces and terrain. We'll explain various drivetrain components and cover concepts such as gear ratios/inches, gear range and cadence. This session is open to anyone who is keen to understand these concepts at least a little deeper than just \"I have a '24-speed' bike\", which is something you hear often but literally means nothing!Meeting time: 2:45 pmSession Duration: 2 hoursBlack coffee and snacks are on the house!\n{}",
+    "url": "https://www.pedalintandem.com/experiences/workshops/workshop-drivetrains-and-gear-ratios",
+    "keywords": "workshops"
+  },
+  {
+    "@type": "SportsEvent",
+    "name": "The Games We Play - Badminton",
+    "location": {
+      "postalAddress": "837/1, 2nd Cross, 7th Main, 2nd Stage, Indiranagar, Bengaluru, Karnataka",
+      "@type": "Place",
+      "address": "game theory at caurvery school, indiranagar"
+    },
+    "offers": {
+      "priceCurrency": "INR",
+      "@type": "offers",
+      "badminton meet-up": "250.00",
+      "addOn": {
+        "@type": "offers"
+      }
+    },
+    "dates": [
+      {
+        "startDate": "2024-09-11T19:00:00+05:30",
+        "endDate": "2024-09-11 21:00:00+05:30",
+        "availabilityStarts": "2024-09-11T13:30:00+05:30"
+      }
+    ],
+    "duration": 2,
+    "description": "Relive the joys of childhood games, and remember what it's like to not to take things too seriously, and just embrace the playful, spontaneous and carefree aspects of life. This one involves a few rounds of badminton while you get to meet and bond with members of the community.\nLocation: Game Theory at Caurvery School, Indiranagar - Time: 7 to 9 pm\nIf you don't have badminton shoes or rackets, you can rent them at the venue, from Game Theory.\n{}",
+    "url": "https://www.pedalintandem.com/experiences/thePITStop/the-games-we-play-badminton",
+    "keywords": "thePITStop"
+  },
+  {
+    "@type": "SportsEvent",
+    "name": "Cycle School",
+    "location": {
+      "postalAddress": "837/1, 2nd Cross, 7th Main, 2nd Stage, Indiranagar, Bengaluru, Karnataka",
```

Comment:
there is no `postalAddress` key in https://schema.org/Place.

All events in their indiranagar workshop should have `Place` should have a `name: Pedal In Tandem`, and a `address`.

Did you validate your schema?

Reply:
Yes, I did. it didn't show me any errors or warning.

<img width="1680" alt="Screenshot 2024-09-10 at 2 54 50 PM" src="https://github.com/user-attachments/assets/e146b432-35de-420b-84e8-37b8d79dfb8f">

I added context now and it shows me the error. Fixing it.